runtime/src/main/java/com/flipkart/flux/controller/WorkFlowExecutionController.java

```diff
@@ -237,6 +242,7 @@ private void executeStates(StateMachine stateMachine, Set<State> executableState
                 ActorRef router = this.routerRegistry.getRouter(routerName);
 
                 router.tell(msg, ActorRef.noSender());
+                metricsClient.incCounter(msg.getTaskName());
```

Comment:
keep the counter name in the below format

statemachine.<sm_name>.task.<task_name>.queuesize